test_regress/t/t_class_extends_alias_unsup.out

```diff
@@ -0,0 +1,5 @@
+%Error-UNSUPPORTED: t/t_class_extends_alias_unsup.v:22:22: Unsupported: TYPEDEF 'foo_t' in AstClassExtends
```

Comment:
I am not sure about it. Grammar allows only for `class_type`, but this nonterminal symbol may be resolved to `identifier`, and that type alias is identifier. So the code in this testcase is grammatically correct. I couldn't find any other information about such case.
If only a class reference might be used in `extends` statement, it would be illegal to use a parameter identifier, but such code exists in UVM.
So I am not sure if it is correct, but I think that it may be, so I marked it as unsupported.